minethd.cpp

```diff
@@ -32,7 +32,7 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
-	SetThreadAffinityMask(h, 1ULL << cpu_id);
+	SetThreadAffinityMask((HANDLE)h, 1 << cpu_id);
```

Comment:
HANDLE, like all windows stuff is defined in windows.h 
This cast is not needed for MSVC ( https://msdn.microsoft.com/en-us/library/hh920601.aspx#thread__native_handle_method ) not sure about gcc.